db/migrations/20180716095453-createSlackIntegrationTable.js

```diff
@@ -1,50 +0,0 @@
-module.exports = {
-  up: (queryInterface, DataTypes) => {
-    return queryInterface.createTable('slack_integrations', {
-      id: {
-        allowNull: false,
-        autoIncrement: true,
-        primaryKey: true,
-        type: DataTypes.INTEGER,
-      },
-      slack_channel: {
-        allowNull: false,
-        type: DataTypes.STRING,
-      },
-      date: {
```

Comment:
@IAMOTZ , isn't **_date_** a reserved key word? You can maybe consider to rename it to a more descriptive name